util/virtual_group.go

```diff
@@ -32,3 +34,17 @@ func TotalStakingStoreSizeOfGVG(gvg *virtualgrouptypes.GlobalVirtualGroup, staki
 	}
 	return total.Uint64()
 }
+
+// IsServingAsSecondarySP return whether current sp is one of the object gvg's secondary sp and its index within GVG(if is)
+func IsServingAsSecondarySP(ctx context.Context, client *gfspclient.GfSpClient, selfSpId uint32, bucketId uint64, lvgId uint32) (int, bool, error) {
```

Comment:
fix `selfSpId, bucketId, lvgId` as `selfSpID, bucketID, lvgID`. Recommend using ID in golang.